test/controllers/lawyers_controller_test.rb

```diff
@@ -7,34 +7,36 @@ class LawyersControllerTest < ActionController::TestCase
     assert_response :ok
   end
 
-  test 'POST create makes an unapproved lawyer with a random password' do
+  test 'POST create makes an unapproved lawyer with a random password and notifies lawyer' do
     password = 'blabla'
 
     assert_difference 'Lawyer.count', +1 do
-      post(:create, lawyer: {
-        email: 'bouke@test.nl',
-        password: password,
-        approved: true,
-        name: 'Bouke van der Bijl',
-        address1: 'Netherlands 1',
-        city: 'Amsterdam',
-        province: 'North Holland',
-        telephone: '0612123817',
-        employer_name: 'Zijne Majesteit',
-        employer_address: 'Oranjestraat 1',
-        employment_type: 'Court',
-        practicing: '1',
-        year_of_call: '1900',
-        law_society: 'LSUC',
-        areas_of_practice: ['Environment'],
-        language_of_practice: 'Dutch',
-        experience_with_refugee_sponsorships: '0',
-        experience_with_refugee_sponsorships_clarification: '',
-        insurance: 'No',
-        can_accomodate_meetings: '1',
-        areas_of_interest: ['Providing ongoing assistance to sponsors through SSP matching'],
-        comments: ''
-      })
+      assert_difference 'ActionMailer::Base.deliveries.size', +1 do
+        post(:create, lawyer: {
+          email: 'bouke@test.nl',
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.